### PR TITLE
Add db seed data for development

### PR DESCRIPTION
**Pull request by appleJax**
Added the faker gem and some seed data so we can have a non-empty database for development.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,59 @@
...
+    timeslot = Timeslot.create!({
+      event_id: event.id,
+      start_time: Faker::Time.forward(30)
+    })
+
+    event_users.each do |user|
+      Preference.create!({
+        timeslot_id: timeslot.id,
+        user_id: user.id,
+        preference_type: rand(4)
```
Comment (gabrielbonner) [inline db/seeds.rb]: @appleJax This could be `rand(5)` to include the blue/preferred option as well.

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,59 @@
...
+end
+
+NUM_EVENTS.times do
+  location = Location.create!({
+    name: Faker::Company.unique.name,
+    address: Faker::Address.unique.street_address
+  })
+
+  event = Event.create!({
+    name: Faker::GameOfThrones.unique.dragon,
```
Comment (gabrielbonner) [inline db/seeds.rb]: Nice one 😄

**Comment by gabrielbonner** [comment]
@appleJax Looks good to me 👍

**Comment by appleJax** [comment]
@gabrielbonner I made some minor changes. Separated out `NUM_LOCATIONS` into its own constant instead of creating a new location for every event. Also, changed some `Faker` data to make it a little more readable. Feel free to merge after you give it another once-over!

**Pull request merged**